### ISSUE TITLE
exec_start / exec_stop

### ISSUE THREAD

**Issue by sharky1337** [cbsd/cbsd#193]
Hello ,

I'm not able to add or change these values via cbsd jconfig .

/usr/jails/jailctl/jsetup-tui: get_construct_exec_start: not found

Is this a expected behavior ?

**Comment by olevole** [comment]
Hi,

This is definitely wrong behavior. Can you share this info:

1) cbsd version   ( cbsd -c version )
2) installation method ( from the ports, pkg or GitHub )
3) This is fresh installation or upgrade

Also if the version of CBSD is not 11.1.4 (latest), can you update **/usr/ports** tree and reinstall CBSD to latest 11.1.4:

~~~
% make -C /usr/ports/sysutils/cbsd clean
% make -C /usr/ports/sysutils/cbsd reinstall
% cbsd initenv
~~~

**Comment by sharky1337** [comment]
1. # cbsd -c version
11.1.4
2. ports
3. fresh

**Comment by olevole** [comment]
Ahh, got it. 
It seems that I have never implemented editing of these settings) I will correct in the next two-three days.

Keep in mind, if you want to add some extra of your command on jail start, it's better to use  ~cbsd/jails-system/ directory: https://www.bsdstore.ru/en/11.1/wf_jconfig_ssi.html#execscript

Ok, thanks for feedback/issue.

**Comment by olevole** [comment]
Fixed in 11.1.5.
Thanks for contribution.
